src/CryptoAxus.Application/Dto/AuctionDto.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace CryptoAxus.Application.Dto;
+
+public class AuctionDto : BaseDto
+{
+    public string? NftId { get; set; }
+
+    public string? NftCopyId { get; set; }
+
+    public int? Quantity { get; set; }
+
+    public decimal? MinimumBid { get; set; }
+
+    public bool? SoftDelete { get; set; }
+
+    public DateTime? StartingDateTime { get; set; }
+
+    public DateTime? EndingDateTime { get; set; }
+
+    public AuctionDto()
+    {
+    }
+}
```

Comment:
Add another constructor that takes parameters for each property and assigns values to it.

Reply:
Resolved